README.md

```diff
@@ -94,36 +94,55 @@ Please change the directory after cloning the repository using the `cd <folder-n
 
 ### ⬇️ Install Dependencies
 
-Next, install the dependencies by running the following command in the `react-play` repo:
+Next, install the dependencies by running the following command in the `react-play` repo. we recommand using `yarn` but you can install using `npm` too
 
 ```bash
-npm install
+yarn install
 ```
 
 Or
-
-```bash
-yarn install
+```
+npm install
 ```
 
+if you donot have `yarn` installed on your pc follow to below steps to install it.
+
+**Windows**
+1. open your command promt as administrator.
+2. write `corepack enable` and hit enter.
+3. then `npm install --global yarn`
+
+**Linux**
+1. open terminal and hit `npm install --global yarn`
+
+**MacOS**
+1. open terminal and hit `npm install --global yarn`
+or
+`brew install yarn`
+
+**Or Download Package**
+if you are unable to install yarn following the above mention process then you can simply download the package and install it. Visit the official link of yarpkg there you can on the website just expand the `Alternative` section and it will ask for the version to download for windows/linux/mac.
```

Comment:
If you are unable to install yarn following the above-mentioned process, then you can simply download the package and install it. Visit the official website of Yarn; there you can just expand the "Alternative" section and it will ask for the version to download for Windows, Linux, or Mac.

Reply:
added